### PR TITLE
Registration update

### PR DESCRIPTION
**Pull request by dzenanz**

### CODE REVIEW

File: 5_Registration.ipynb [hunk 1]
```diff
@@ -11,17 +11,6 @@
     "# Image Registration"
```
Comment (thewtex) [inline 5_Registration.ipynb:11]: <p>`itk.GetArrayViewFromImage` -&gt; `itk.array_view_from_image`</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (thewtex) [inline 5_Registration.ipynb:11]: <p>`itk.GetArrayViewFromImage` -&gt; `itk.array_view_from_image`</p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (thewtex) [inline 5_Registration.ipynb:11]: <p>Anwer -&gt; Answer</p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (thewtex) [inline 5_Registration.ipynb:11]: <p>Anwer -&gt; Answer</p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (thewtex) [inline 5_Registration.ipynb:11]: <p>`itk.GetArrayViewFromImage` -&gt; `itk.array_view_from_image`</p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (thewtex) [inline 5_Registration.ipynb:11]: <p>`itk.GetArrayViewFromImage` -&gt; `itk.array_view_from_image`</p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (thewtex) [inline 5_Registration.ipynb:11]: <p>`itk.GetArrayViewFromImage` -&gt; `itk.array_view_from_image`</p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

**Comment by thewtex** [comment]
Looking good

**Comment by dzenanz** [comment]
Merging the ready part now.

**Pull request merged**